e2e/plugin-typescript-e2e/tests/collect.e2e.test.ts

```diff
@@ -0,0 +1,76 @@
+import { cp } from 'node:fs/promises';
+// eslint-disable-next-line unicorn/import-style
+import path, { join } from 'node:path';
```

Comment:
For consistency with the rest of the codebase, could you use `path.join`, please?

```suggestion
import path from 'node:path';
```